content/applications/hr/recruitment/schedule_interviews.rst

```diff
@@ -0,0 +1,325 @@
+   For new applicants who are new to the :guilabel:`First Interview` stage, this is the default.
+
+   If there is one meeting already scheduled, the smart button displays :guilabel:`1 Meeting`, with
+   the date of the upcoming meeting beneath it. If more than one meeting is scheduled, the button
+   displays :guilabel:`Next Meeting`, with the date of the first upcoming meeting beneath it.
+
+Doing so loads the *Calendar* application, showing the scheduled meetings and events for the
+currently signed-in user, as well as the employees who are listed under the :guilabel:`Attendees`
+section on the right side of the calendar view. To change the currently loaded meetings and events
+being displayed, uncheck an attendee whose calendar events are to be hidden. Only the checked
+attendees are visible on the calendar.
+
+The default view is the :guilabel:`Week` view. To change the calendar view, click the
+:guilabel:`Week` button, then select the desired view from the drop-down menu. The other options are
+either :guilabel:`Day`, :guilabel:`Month`, or :guilabel:`Year`.
+
+An option to display or hide weekends is available. Click the :guilabel:`Week` button, then click
+:guilabel:`Show weekends` to deactivate it (the default is to show weekends). Check :guilabel:`Show
+weekends` to display weekend dates.
+
+To change the displayed date range for the calendar, either use the :icon:`fa-arrow-left`
+:guilabel:`(left arrow)`, :icon:`fa-arrow-right` :guilabel:`(right arrow)`, or :guilabel:`Today`
+buttons above the calendar, or click on a date in the calendar on the right side of the displayed
+calendar.
+
+.. image:: schedule_interviews/calendar.png
+   :align: center
+   :alt: The calendar view, highlighting how to change the displayed meetings.
+
+To add a meeting to the calendar when in the day or week view, click on the start time of the
+meeting and drag to the end time, to select the date, time, and the length of the meeting. A meeting
+can also be added in this view by clicking on the day *and* the time slot the meeting is to take
+place.
+
+Both methods cause a :ref:`New Event <schedule_interviews/event-card>` pop up window to appear.
+
+.. _schedule_interviews/event-card:
+
+New event pop-up window
+-----------------------
+
+Clicking a grid corresponding with the time and date opens the :guilabel:`New Event` pop-up window
+to schedule a meeting. Enter the information on the form. The only required fields to enter are the
+:guilabel:`Meeting Title`, and the :guilabel:`Start` and :guilabel:`End` fields. Once the card
+details are entered, click :guilabel:`Save & Close` to save the changes and create the interview.
+
+The fields available to populate or modify on the :guilabel:`New Event` card are as follows:
+
+- :guilabel:`Meeting Title`: enter the subject for the meeting. This should clearly indicate the
+  purpose of the meeting. The default subject is the :guilabel:`Subject/Application Name` on the
+  applicant's card.
```

Comment:
This is always a hard call because it's a huge/big/important field and I think it should have a name for the field, but it doesn't. So I went with this to have every bullet point start with bold.